packages/features/insights/server/insightsDateUtils.ts

```
@@ -0,0 +1,206 @@
+}: {
+  start: dayjs.Dayjs;
+  end: dayjs.Dayjs;
+  timeView: TimeViewType;
+  wholeStart: dayjs.Dayjs;
+  wholeEnd: dayjs.Dayjs;
+}): string => {
+  const omitYear = wholeStart.year() === wholeEnd.year();
+
+  switch (timeView) {
+    case "day":
+      const shouldShowMonth = wholeStart.isSame(start, "day") || start.date() === 1;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)